test/distributed/test_functional_api.py

```diff
@@ -429,92 +444,61 @@ def allred_mesh_dim(input):
         )
 
 
-BACKEND = dist.Backend.NCCL if torch.cuda.is_available() else dist.Backend.GLOO
-WORLD_SIZE = 2
+BACKEND = (
+    dist.Backend.HCCL
+    if TEST_HPU
+    else (dist.Backend.NCCL if torch.cuda.is_available() else dist.Backend.GLOO)
+)
```

Comment:
For better readability, can we keep the original line, and just do conditional overwrite?
```
BACKEND = dist.Backend.NCCL if torch.cuda.is_available() else dist.Backend.GLOO

if TEST_HPU:
    BACKEND = dist.Backend.HCCL
elif ...
```

Reply:
Done in commit : 7e6c741b9efe19c7d3fc0d6326becec6879c98c9